app/views/movies/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h3><%= @movie.title %></h3>
+<h5><%= @movie.creation_year %></h5>
+<h5><%= @movie.genre %></h5>
+
+<section class= "Actors from Youngest to Oldest">
+  <h3>"Actors from Youngest to Oldest"</h3>
+  <% @movie.youngest_to_oldest.each do |actor| %>
+    <ul><li><%= actor.name %></li></ul>
+    <ul><li><%= actor.age %></li></ul>
+  <% end %>
+</section>
+
+<section class= "Average Age of Actors">
+  <h3>"Average Age of Actors"</h3>
+  <p> <%= @movie.average_age %></p>
+</section>
+
+<p><%= link_to "Add an actor to this movie", "/movies/#{@movie.id}/actors/new" %></p>
```

Comment:
Grr, not what the story is asking for (but it does work, so I can give you credit)